receiver/mongodbreceiver/testdata/integration/expected.3_0.json

```diff
@@ -43,23 +43,39 @@
                         "dataPoints": [
                            {
                               "asInt": "2",
-                              "startTimeUnixNano": "1659372711430846000",
-                              "timeUnixNano": "1659372771440281000"
+                              "startTimeUnixNano": "1660317989210380000",
+                              "timeUnixNano": "1660318049220463000"
                            }
                         ]
                      },
                      "unit": "{databases}"
                   },
+                  {
+                     "description": "The time the global lock has been held.",
+                     "name": "mongodb.global_lock.time",
+                     "sum": {
+                        "aggregationTemporality": "AGGREGATION_TEMPORALITY_CUMULATIVE",
+                        "dataPoints": [
+                           {
+                              "asInt": "62216",
+                              "startTimeUnixNano": "1660317989210380000",
+                              "timeUnixNano": "1660318049220463000"
+                           }
+                        ],
+                        "isMonotonic": true
+                     },
+                     "unit": "ms"
+                  },
```

Comment:
I'm guessing this corresponds to removal of this metric from the readme's version-specific notes.

Reply:
Before when collecting global lock time, it was checking for version 4.0+ in the metric path `"globalLock", "totalTime"`. Else it would use the metric path `locks", ".", "timeAcquiringMicros"` but that path does not exist in 3.0 which was causing partial errors. This metric can be collected for 3.0 as shown in the pr description which is why `expected.3_0.json` got updated.